examples/fullstack/frontend/build.yaml

```diff
@@ -0,0 +1,9 @@
+# Read about `build.yaml` at https://pub.dev/packages/build_config
+# To update generated code, run `pub run build_runner build`
+targets:
+  $default:
+    builders:
+      source_gen|combining_builder:
+        options:
+          ignore_for_file:
+          - prefer_expression_function_bodies
```

Comment:
Ugh. So. This exists because we have a repo-wide link to `prefer_expression_function_bodies`.

json_serializable doesn't always do that, so we need to ignore here to make the linter happy.

Hrm...